index.js

```diff
@@ -169,14 +169,35 @@ exports.login = async function (options) {
 };
 
 exports.viewAccount = async function (options) {
-    let near = await connect(options);
-    let account = await near.account(options.accountId);
-    let state = await account.state();
-    if (state && state.amount) {
-        state['formattedAmount'] = utils.format.formatNearAmount(state.amount);
+    try {
+        let near = await connect(options);
+        let account = await near.account(options.accountId);
+        let state = await account.state();
+        if (state && state.amount) {
+            state['formattedAmount'] = utils.format.formatNearAmount(state.amount);
+        }
+        console.log(`Account ${options.accountId}`);
+        console.log(inspectResponse.formatResponse(state));
+    } catch(e) {
+        console.log(chalk`\n{bold.red Account {bold.white ${options.accountId}} is not found in {bold.white ${config.helperAccount}} network.\n}`);
+
+        const prefix = String(options.accountId).match(/[^\.]*$/gi)[0];
```

Comment:
nit: seems to be suffix, not prefix

Reply:
I just called it as it's been described in issue description, will rename np